### PR TITLE
return sender ids instead of column names

### PR DESCRIPTION
**Pull request by wochinge**
**Proposed changes**:
-  return distinct list of sender ids instead of column names

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [x] updated the changelog
- [x] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa_nlu#code-style) for instructions)

### CODE REVIEW

**Comment by wochinge** [comment]
@MetcalfeTom The KeyView is lazy in comparison to a `list`. Not sure how performance critical that is. We could also change the type in general to `Iterable`?

File: .travis.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ install:
   - python -m pip install -U pip
   - pip install git+https://github.com/tmbo/MITIE.git
   - pip install -r requirements-dev.txt
-  - pip install -e .
+  - pip install --no-use-pep517 -e .
```
Comment (wochinge) [inline .travis.yml]:
```suggestion
  - pip install -e .
```

**Comment by wochinge** [comment]
@MetcalfeTom ?

**Comment by MetcalfeTom** [comment]
Tested it out and `dictkeys` is slightly quicker at iterating, so let's go with your suggestion of changing the type in the base class

**Comment by wochinge** [comment]
@MetcalfeTom Are you taking this over or should I finish it? You have to change the datatype in all keys() functions

**Pull request merged**